### PR TITLE
feat(ui): Add new loading pattern logo

### PR DESCRIPTION
**Pull request by eburairu**
New loading pattern logo and fixed logo added. 👨‍🎨

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Comment by eburairu** [comment]
I like this one.

File: datahub-web-react/src/images/datahub-logo-color-loading_pendulum.svg [hunk 1]
```diff
@@ -15,49 +15,53 @@
         </linearGradient>
 
         <mask id="red_mask">
-            <circle cx="90" cy="90" r="34" fill="white" />
-            <circle cx="90" cy="90" r="26" fill="black" />
-            <rect width="34" height="68" x="56" y="56" fill="black" />
-            <circle cx="90" cy="60" r="4" fill="white" />
-            <circle cx="90" cy="120" r="4" fill="white" />
+            <circle r="34" fill="white" />
```
Comment (jjoyce0510) [inline datahub-web-react/src/images/datahub-logo-color-loading_pendulum.svg:18]: This one is my favorite :)

File: datahub-web-react/src/images/datahub-logo-color-loading_simple.svg [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<svg id="datahub-logo-color_loading_panic" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
```
Comment (jjoyce0510) [inline datahub-web-react/src/images/datahub-logo-color-loading_simple.svg:1]: Also really like this one!!!

**Review by jjoyce0510 — APPROVED** [review]
These are amazing. Great contribution @eburairu!

Thank you on behalf of the core team 😄

File: datahub-web-react/src/images/datahub-logo-color-loading_pendulum.svg (hunk 1 above)
Comment (eburairu) [reply datahub-web-react/src/images/datahub-logo-color-loading_pendulum.svg:18]: How about making a watch with a red arc for the second hand, a yellow arc for the minute hand, and a blue arc for the hour hand? ⌚ 
DataHub logo would point to 1:40. 😄

**Review by eburairu — COMMENTED** [review]
datahub-logo-color-changelog

Change angles for matching to the original logo.
- Red arc: 180 degree → 160 degree
- Yellow arc: 306 degree → 310 degree
- Blue arc: 216 degree → 215 degree
- Joint: 60 degree → 57 degree

File: datahub-web-react/src/images/datahub-logo-color-loading_pendulum.svg (hunk 1 above)
Comment (eburairu) [reply datahub-web-react/src/images/datahub-logo-color-loading_pendulum.svg]: like this 🕐 
datahub-logo-color-loading_clock

**Pull request merged**